beetmoverscript/data/maven_beetmover_task_schema.json

```diff
@@ -0,0 +1,97 @@
+{
+    "title": "Taskcluster beetmover task maven schema",
+    "type": "object",
+    "properties": {
+        "dependencies": {
+            "type": "array",
+            "minItems": 1,
+            "uniqueItems": true,
+            "items": {
+                "type": "string"
+            }
+        },
+        "payload": {
+            "type": "object",
+            "properties": {
+                "upload_date": {
+                    "type": "number"
+                },
+                "build_number": {
+                    "type": "number"
+                },
+                "locale" : {
+                    "type" : "string"
+                },
+                "maxRunTime" : {
+                    "type" : "number"
+                },
+                "version": {
+                    "type": "string"
+                },
+                "next_version" : {
+                     "type" : "string"
+                },
+                "appVersion": {
+                    "type": "string"
+                },
+                "releaseProperties" : {
+                    "type" : "object",
+                    "properties" : {
+                        "appName": {
+                            "type" : "string"
+                        },
+                        "buildid": {
+                            "type" : "string"
+                        },
+                        "appVersion": {
+                            "type" : "string"
+                        },
+                        "hashType":  {
+                            "type" : "string"
+                        },
+                        "platform":  {
+                            "type" : "string"
+                        },
+                        "branch":  {
+                            "type" : "string"
+                        }
+                    },
+                    "required" : ["appName"]
+                },
+                "upstreamArtifacts": {
+                    "type": "array",
+                    "items": {
+                        "type": "object",
+                        "properties": {
+                            "taskType": {
+                                "type": "string"
+                            },
+                            "locale": {
```

Comment:
Nit: we don't need locale, do we?

Reply:
It can still be present in beetmover-geckoview jobs coming from [here](https://hg.mozilla.org/mozilla-central/file/tip/taskcluster/taskgraph/transforms/beetmover_geckoview.py#l91).

Reply:
IIUC, this line is for `payload.locale` instead of `payload.upstreamArtifacts[0].locale`

Reply:
You're right, don't mind me!